python/pyspark/ml/tests/connect/test_legacy_mode_classification.py

```diff
@@ -217,7 +246,15 @@ def test_save_load(self):
 
             fs_model_path = os.path.join(tmp_dir, "fs", "model")
             model.save(fs_model_path)
-            loaded_model = LORV2Model.load(fs_model_path)
+
+            with torch.serialization.safe_globals(
```

Comment:
If torch>=2.6.0 is used, then it is up to the customer to specify the safe_globals.